### PR TITLE
Adding a va-nav-category selector for piano keys. Closes #1787

### LINKED ISSUE #1787: Refactor piano keys into a reusable pattern tied to class names
Currently piano key styles are using `#content .main .section.one ul[class*="block-grid-"]` into as a selector. That's longer than it needs to be, and causes all kinds of headaches for other `ul` elements with `#content` or `.main` as parents.

Can refactor into a ~~`.va-nav-pianokeys`~~ `.va-nav-category` pattern.

### PR DESCRIPTION
**Pull request by webinista**
Blocks #1813, #1814, #1816, #1817, #1819, #1820, #1821.  

It's the same change, but to almost every file on the site, which is why I split them into several pull requests.

### CODE REVIEW

File: _sass/components/_va-nav-category.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  
+  @media #{$small} {
+    margin: 0 -0.625rem !important;
+  }
+  
+  &::before, &::after {
+    clear: both;
+    content: " ";
+    display: table; 
+  }
```
Comment (crwallace) [inline _sass/components/_va-nav-category.scss]: I am not sure which browsers we are trying to support be if we need to support IE6+ then we might want to add `zoom` to trigger `hasLayout`.

File: _sass/components/_va-nav-category.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      background: rgba(0,0,0,0.1);
+      border-left-color: $color-primary;
+      text-decoration: none;
+    }
+  }
+}  
+
+.va-nav-category-title, 
+.va-nav-category h5 {
+  color: #112e51;
```
Comment (crwallace) [inline _sass/components/_va-nav-category.scss]: Can we make this color a variable in case other people need to user it somewhere else in the site?

**Comment by crwallace** [comment]
Just a couple of questions on this, but other than that it LGTM!

File: _sass/components/_va-nav-category.scss (hunk 1 above)
Comment (webinista) [inline _sass/components/_va-nav-category.scss]: I believe we're supporting IE8+.

Comment (crwallace) [inline _sass/components/_va-nav-category.scss]: Cool! Looks good then!

**Pull request merged**